.github/workflows/multibuild.yaml

```diff
@@ -24,23 +24,33 @@ jobs:
         include:
           - os: ubuntu-latest
             output-name: sshnp-linux-x64
+            ext: ''
           - os: macOS-latest
             output-name: sshnp-macos-x64
+            ext: ''
           - os: windows-latest
             output-name: sshnp-windows-x64
+            ext: '.exe'
 
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
       - uses: dart-lang/setup-dart@b64355ae6ca0b5d484f0106a033dd1388965d06d # v1.6.0
       - run: mkdir sshnp
       - run: mkdir tarball
+      - if: ${{ matrix.os != 'windows-latest' }}
+        run: mkdir sshnp/debug
       - run: dart pub get
```

Comment:
did a separate mkdir instead of mkdir -p because I'm fairly confident that the powershell mkdir doesn't support that flag.